plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -820,9 +825,11 @@ protected String doBuildImage(String repository,
                                                             .method("POST")
                                                             .path("/build")
                                                             .query("rm", 1)
-                                                            .query("pull", 1)
```

Comment:
As far as i know `true` may be used as query parameter value as well as 1, so you can just put `.query("pull", doForcePull);`